cmd/snake-runner/scheduler.go

```diff
@@ -70,13 +74,44 @@ func (snake *Snake) startScheduler() error {
 			)
 		}
 
+		err = sidecar.NewShellSidecarBuilder().
+			Spawner(spawn).
+			Build().
+			CheckPrerequisites(ctx)
+		if err != nil {
+			return karma.Format(
+				err,
+				"prerequisites check failed while running snake-runner with SNAKE_EXEC_MODE=shell;"+
+					" make sure that specified binaries are installed",
+			)
```

Comment:
We also need to include a link on the installation instructions here on snake-ci.com/docs (we don't have it yet)

Reply:
It can be added later, but we definitely should do it before the release.